src/pytorch_lightning/profiler/__init__.py

```diff
@@ -28,4 +29,5 @@
     "PyTorchProfiler",
     "SimpleProfiler",
     "XLAProfiler",
+    "HPUProfiler",
```

Comment:
let's not add it here. this is deprecated sub-dir

Reply:
I got mypy / flake errors